From 85479a21ea01ca8582e2e01f551fd58dc499703c Mon Sep 17 00:00:00 2001
From: Mark Reid <mreid@mozilla.com>
Date: Mon, 5 Jan 2015 10:04:33 -0400
Subject: [PATCH] Increase message size limit from 64KB to 8MB

---
 message/message.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/message/message.go b/message/message.go
index 4b87f91..b0b0827 100644
--- a/message/message.go
+++ b/message/message.go
@@ -33,7 +33,7 @@ const (
 	HEADER_DELIMITER_SIZE = 2                         // record separator + len
 	HEADER_FRAMING_SIZE   = HEADER_DELIMITER_SIZE + 1 // unit separator
 	MAX_HEADER_SIZE       = 255
-	MAX_MESSAGE_SIZE      = 64 * 1024
+	MAX_MESSAGE_SIZE      = 8 * 1024 * 1024
 	MAX_RECORD_SIZE       = HEADER_FRAMING_SIZE + MAX_HEADER_SIZE + MAX_MESSAGE_SIZE
 	RECORD_SEPARATOR      = uint8(0x1e)
 	UNIT_SEPARATOR        = uint8(0x1f)
-- 
1.9.4 (Apple Git-50.2)

