From 70654e1d8f917f5e97a8305dd5a632ebe086d252 Mon Sep 17 00:00:00 2001
From: Mark Reid <mreid@mozilla.com>
Date: Mon, 12 Jan 2015 09:25:48 -0400
Subject: [PATCH] Add cmdline tool for uploading to S3.

---
 CMakeLists.txt          |   8 +++
 1 file changed, 8 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 868bf50..602deac 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,6 +36,7 @@ set(SBMGRLOAD_EXE "${PROJECT_PATH}/bin/heka-sbmgrload${CMAKE_EXECUTABLE_SUFFIX}"
 set(INJECT_EXE "${PROJECT_PATH}/bin/heka-inject${CMAKE_EXECUTABLE_SUFFIX}")
 set(LOGSTREAMER_EXE "${PROJECT_PATH}/bin/heka-logstreamer${CMAKE_EXECUTABLE_SUFFIX}")
 set(HEKA_CAT_EXE "${PROJECT_PATH}/bin/heka-cat${CMAKE_EXECUTABLE_SUFFIX}")
+set(HEKA_EXPORT_EXE "${PROJECT_PATH}/bin/heka-export${CMAKE_EXECUTABLE_SUFFIX}")
 
 option(INCLUDE_SANDBOX "Include Lua sandbox" on)
 option(INCLUDE_MOZSVC "Include the Mozilla services plugins" on)
@@ -217,6 +218,13 @@ WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
 
 install(PROGRAMS "${HEKA_CAT_EXE}" DESTINATION bin)
 
+add_custom_target(heka-export ALL
+${GO_EXECUTABLE} install ${LDFLAGS} github.com/mozilla-services/heka/cmd/heka-export
+DEPENDS hekad
+WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
+
+install(PROGRAMS "${HEKA_EXPORT_EXE}" DESTINATION bin)
+
 add_custom_target(sbmgr ALL
 ${GO_EXECUTABLE} install ${LDFLAGS} github.com/mozilla-services/heka/cmd/heka-sbmgr
 DEPENDS hekad)
