From b31a2ce9ab6d3f5cf744c8234fd145ae5c14a786 Mon Sep 17 00:00:00 2001
From: Mark Reid <mreid@mozilla.com>
Date: Wed, 4 Feb 2015 17:10:10 -0400
Subject: [PATCH] Update build to include heka-s3list and heka-s3cat

---
 CMakeLists.txt | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a5cdd21..705d223 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -37,6 +37,9 @@ set(INJECT_EXE "${PROJECT_PATH}/bin/heka-inject${CMAKE_EXECUTABLE_SUFFIX}")
 set(LOGSTREAMER_EXE "${PROJECT_PATH}/bin/heka-logstreamer${CMAKE_EXECUTABLE_SUFFIX}")
 set(HEKA_CAT_EXE "${PROJECT_PATH}/bin/heka-cat${CMAKE_EXECUTABLE_SUFFIX}")
 set(HEKA_EXPORT_EXE "${PROJECT_PATH}/bin/heka-export${CMAKE_EXECUTABLE_SUFFIX}")
+set(HEKA_S3LIST_EXE "${PROJECT_PATH}/bin/heka-s3list${CMAKE_EXECUTABLE_SUFFIX}")
+set(HEKA_S3CAT_EXE "${PROJECT_PATH}/bin/heka-s3cat${CMAKE_EXECUTABLE_SUFFIX}")
+set(GET_CLIENTS_EXE "${PROJECT_PATH}/bin/get-clients${CMAKE_EXECUTABLE_SUFFIX}")
 
 option(INCLUDE_SANDBOX "Include Lua sandbox" on)
 option(INCLUDE_MOZSVC "Include the Mozilla services plugins" on)
@@ -225,6 +228,27 @@ WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
 
 install(PROGRAMS "${HEKA_EXPORT_EXE}" DESTINATION bin)
 
+add_custom_target(heka-s3list ALL
+${GO_EXECUTABLE} install ${LDFLAGS} github.com/mozilla-services/heka/cmd/heka-s3list
+DEPENDS hekad
+WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
+
+install(PROGRAMS "${HEKA_S3LIST_EXE}" DESTINATION bin)
+
+add_custom_target(heka-s3cat ALL
+${GO_EXECUTABLE} install ${LDFLAGS} github.com/mozilla-services/heka/cmd/heka-s3cat
+DEPENDS hekad
+WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
+
+install(PROGRAMS "${HEKA_S3CAT_EXE}" DESTINATION bin)
+
+add_custom_target(get-clients ALL
+${GO_EXECUTABLE} install ${LDFLAGS} github.com/mozilla-services/heka/cmd/get-clients
+DEPENDS hekad
+WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
+
+install(PROGRAMS "${GET_CLIENTS_EXE}" DESTINATION bin)
+
 add_custom_target(sbmgr ALL
 ${GO_EXECUTABLE} install ${LDFLAGS} github.com/mozilla-services/heka/cmd/heka-sbmgr
 DEPENDS hekad)
-- 
1.9.4 (Apple Git-50.2)

